pkgdb2/forms.py

```diff
@@ -409,3 +409,11 @@ def __init__(self, *args, **kwargs):
                 (status, status)
                 for status in kwargs['status']
             ]
+
+
+class UnretireForm(BranchForm):
+    """ Form to ask for a package to be un-retired. """
+    review_url = wtforms.TextField(
+        'review_url',
+        [wtforms.validators.optional()],
```

Comment:
Would it be an idea to check for requirement of this field when generating this form, so that you can indicate whether it's required?